### PR TITLE
Added base config doc file

### PR DESCRIPTION
**Pull request by amayagil**
##### SUMMARY

Added base config doc file

##### ISSUE TYPE

- Docs Pull Request

##### ADDITIONAL INFORMATION

```paste below

```

### CODE REVIEW

**Comment by amayagil** [comment]
please approve latest changes to file

**Comment by amayagil** [comment]
Fixed conflicts

File: training/02_Getting_Started/02_config_your_env.adoc [hunk 1]
```diff
@@ -0,0 +1,272 @@
+
+= How to set up your laptop to develop AgnosticD workloads [[configs-laptop]]
+
+In this document is aimed to be a reference on how to setup your laptop (both Linux and Mac OS) for Ansible Agnostic Deployer development with OpenStack, AWS and Azure.
+
+Whilst this document primarily at Red Hatters, and partners, who can self provision an OpenStack (OSP) Sandbox via Red Hat’s link:https://rhpds.redhat.com[RHPDS] it may also be useful to others working both with OpenStack and Ansible Agnostic Deployer aka AgnosticD.
+
+For the purpose of this exercise we’ll use the AgnosticD config link:https://github.com/redhat-cop/agnosticd/tree/development/ansible/configs/a-base-config[a-base-config] which is simple to deploy, and extensible for anyone looking to deploy some instances on OpenStack.
+
+I’ll assume you have either self provisioned, or been provided with, an OpenStack Sandbox instance and have the necessary ssh login details etc.
```
Comment (newgoliath) [inline training/02_Getting_Started/02_config_your_env.adoc:10]: Omit.

**Comment by amayagil** [comment]
need something else from me?

**Pull request merged**